dkron/store.go

```diff
@@ -242,6 +244,26 @@ func (s *Store) GetJobs(options *JobOptions) ([]*Job, error) {
 		if options != nil && options.ComputeStatus {
 			job.Status = job.GetStatus()
 		}
+
+		// Filter jobs based on tags
+		if options != nil {
```

Comment:
Since we are already looping on each Result, this shouldn't decrease performance much.